Makefile

```diff
@@ -0,0 +1,22 @@
+SHELL := bash
```

Comment:
I’m not a fan of Makefiles because it isn’t something common on Windows. Personally I don’t want to go back to them because all that information can just be in the drone yml.